docs/core/tools/dotnet-publish.md

```diff
@@ -61,6 +61,10 @@ Specifies the path for the output directory. If not specified, it defaults to *.
 
 Configuration to use when building the project. The default value is `Debug`.
 
+`--manifest <PATH_TO_MANIFEST_FILE>`
```

Comment:
if we're going to add this, should we show syntax for 1.0 vs 2.0? If not, I'd probably wait to add this only when we get the tabs later this month.